CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Changes
 
+## edge-20.7.1
+
+This edge release improves messaging for retries when pods are unschedulable,
+adds a feature to persist prometheus data as an option to storing the data in
+memory, and upgrades the protobuf version to v1.4.2
+
+* Add better messaging during scheduling errors on retry for HA installations
+* Add option to persist prometheus data (thanks @naseemkullah!)
+* Upgrades generated protobuf files to v1.4.2
```

Comment:
Not user-facing, so we can leave this one out

Reply:
+1 to omitting